packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/shippingInformation.css

```diff
@@ -27,6 +27,11 @@
     font-weight: 600;
 }
 
+.editButton {
+    padding: 1rem;
+    margin: -1rem;
```

Comment:
I see this repeated for all the edit buttons - what problem does this solve?

Reply:
It was requested by UX to have padding 16px (1rem) padding for these buttons it . Negative margins compensate this added padding, avoiding change of the the heading row dimensions and button positioning.

Reply:
It increases the clickable area without affecting the alignment.